src/Coypu/Coypu.csproj

```diff
@@ -35,6 +35,7 @@ Copyright © 2018 Adrian Longley and Contributors</Copyright>
     <DocumentationFile>bin\Debug\Coypu.xml</DocumentationFile>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Selenium.WebDriver" Version="4.4.0" />
+    <PackageReference Include="Selenium.WebDriver" Version="4.8.1" />
+    <PackageReference Include="Selenium.WebDriver.MSEdgeDriver" Version="110.0.1587.50" />

```

Comment:
Why do we need the edge driver? This seems wrong as we should not have a dep on a specific driver in coypu itself